cmake/globals.cmake

```diff
@@ -435,15 +433,18 @@ macro(add_stripped_executable _target)
                             COMMAND objcopy --add-gnu-debuglink=${_target}.debug ${_target}
                             COMMENT "Stripping ${_target}")
       elseif(APPLE)
+         
          if(${ARGV1} STREQUAL "MACOSX_BUNDLE")
-            set(STRIP_TARGET "${_target}.app/contents/MacOS/${_target}")
+            set(STRIP_TARGET "${_target}.app/Contents/MacOS/${_target}")
          else()
             set(STRIP_TARGET "${_target}")
          endif()
+         
          add_custom_command(TARGET ${_target} POST_BUILD
                             COMMAND dsymutil -o ./${_target}.dSYM ${STRIP_TARGET}
-                            COMMAND strip ${STRIP_TARGET}
+                            COMMAND strip -x -S ${STRIP_TARGET}
```

Comment:
For reference, from `man strip`:

       -S     Remove the debugging symbol table entries (those created by the -g option to cc(1) and
              other compilers).

       -x     Remove all local symbols (saving only global symbols).

And

       If the executable loads objects, however, the global symbols that the objects reference from the
       executable also must not be stripped.  In this case, when linking the executable you should use
       the `-exported_symbols_list` option of the link editor ld(1) to limit which symbols can be
       referenced by the objects.  Then you only need to strip local and debug symbols, like that:
              % strip -x -S executable

I've tried to follow that advice here and elsewhere in the PR.